### DISCUSSION TITLE
【0x08】Buidl an Voxel Art NFT dApp Based on Scaffold ETH

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#61] [Projects]
Link to L1 Lesson:
> https://github.com/WeLightProject/Web3-dApp-Camp/discussions/19

Video: 
> https://www.youtube.com/watch?v=g0xdkIEysDo

Main Topic:

- How to create an Voxel HTML Show Page by `.vox` File
- How to build a local chain by hardhat in `scaffold-eth`
- How to deploy the contract by hardhat to local chain
- How to start the react dApp in `scaffold-eth`
- mint an Voxel NFT to the Voxel-NFT-Contract by yourself!

**Comment by leeduckgo** [comment]
打卡活动：
根据直播课程或视频进行实践，将最终包含了`Voxel NFT`的本地 dApp 进行截图，在本帖下回复，即表示打卡成功。

**Comment by alertcat** [comment]
![proof](https://user-images.githubusercontent.com/73512451/160828743-726f9b44-71c1-4d12-a596-9ce8013cbd06.png)

**Reply by leeduckgo** [reply]
GooD

**Comment by itherunder** [comment]
![image](https://user-images.githubusercontent.com/61953384/160973756-0fff4f17-14f6-4bfc-ab57-65d45c764cb7.png)
<b>蛮好玩的，哈哈</b>

**Comment by snowyaya** [comment]
![image](https://user-images.githubusercontent.com/75382121/161408911-6b241ac7-b98a-4b2b-b301-41c1623ddb27.png)

![voxel-viewer](https://user-images.githubusercontent.com/75382121/161409152-abcfe0d5-c6d4-467d-beaa-3d9a875461e0.gif)
